account_invoice_refund_link/tests/test_invoice_refund_link.py

```diff
@@ -7,21 +7,20 @@
 from .. import post_init_hook
 
 
-class TestInvoiceRefundLink(common.SavepointCase):
+class TestInvoiceRefundLink(common.TransactionCase):
```

Comment:
Please drop a comment saying that this is specifically TransactionCase because the deadlock.